json_validation/services/document_service.py

```
@@ -0,0 +1,225 @@
+
+        Args:
+            file_path: Path to document
+
+        Returns:
+            Document data
+
+        Raises:
+            DataExtractionError: If document cannot be loaded
+        """
+        if not self._file_repo.exists(file_path):
+            raise DataExtractionError(f"Document not found: {file_path}")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)